internal/clients/api_client.go

```diff
@@ -116,7 +118,11 @@ func ensureTLSClientConfig(config *elasticsearch.Config) *tls.Config {
 }
 
 func (a *ApiClient) GetESClient() *elasticsearch.Client {
-	return a.es
+	return a.elasticsearch
```

Comment:
There's nothing really enforcing the presence of these clients anymore, it's valid to only configure the Kibana endpoint, but attempt to manage ES resources for instance. 

IMO we should update the `Get*Client` funcs to return an error if the client is nil.